pkgs/by-name/os/osmium-tool/package.nix

```
@@ -25,6 +26,18 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-DObqbzdPA4RlrlcZhqA0MQtWBE+D6GRD1pd9U4DARIk=";
   };
 
+  patches = [
+    # https://github.com/osmcode/osmium-tool/issues/276
```

The two patches correspond to the commits of PR Tencent/rapidjson#719. While that PR is linked in the description of the mentioned upstream issue osmcode/osmium-tool#276, I feel the PR should be mentioned here explicitly as a source, e.g.

```suggestion
    # Work around https://github.com/osmcode/osmium-tool/issues/276
    # by applying changes from https://github.com/Tencent/rapidjson/pull/719
```

(… especially as upstream osmium-tool chose another path (replacing rapidjson, see osmcode/osmium-tool#277) for avoiding the problem.)

---

Thanks @das-g . Applied.